gene/schemas.py

```diff
@@ -0,0 +1,298 @@
+"""This module contains data models for representing VICC normalized
+gene records.
+"""
+from typing import Type, List, Optional, Dict, Union, Any
+from pydantic import BaseModel
+from enum import Enum, IntEnum
+
+
+class SymbolStatus(str, Enum):
+    """Define string constraints for symbol status attribute."""
+
+    WITHDRAWN = "withdrawn"
+    APPROVED = "approved"
+
+
+class Gene(BaseModel):
+    """Gene"""
+
+    label: Optional[str]
+    concept_id: str
+    symbol: str
+    previous_symbols: Optional[list]
```

Comment:
I believe that we don't make Lists optional because we want to return an empty list rather than None. I might be wrong though. 
```suggestion
    previous_symbols: List[str]
```